app/bundles/PageBundle/Entity/HitRepository.php

```diff
@@ -232,7 +232,7 @@ public function countVisitors($seconds = 60, $notLeft = false)
      *
      * @param array $options
      *
-     * @return \DateTime
+     * @return \DateTime|null
      */
     public function getLatestHit($options)
```

Comment:
You could use return type from methods `public function getLatestHit($options): ?\DateTime` and delete the comment

Reply:
Changing the signature of the public method would be a breaking change, which I would rather avoid in this case.